Sources/PointFree/Episode/Episode.swift

```diff
@@ -5,44 +5,59 @@ public struct Episode {
   public private(set) var blurb: String
   public private(set) var codeSampleDirectory: String
   public private(set) var exercises: [Exercise]
+  public private(set) var fullVideo: Video?
```

Comment:
some updates to the episode model including a new video model that will replace the sources arrays down below. keeping optional now so that we can fill in the data later, and then delete the deprecated "sources" arrays.